csharp/codeql-extractor.yml

```diff
@@ -1,4 +1,6 @@
 name: "csharp"
+aliases:
+  - "c#"
```

Comment:
Is the alias case sensitive? Will `--language C#` also work?

Reply:
I think the CLI change would make sense. `c#` looks odd, `C#` reads more natural to me. There are other languages (JS, TS) where the short name is like an acronym, so I think the upper case would be the obvious choice there too.